book/src/voluntary-exit.md

```diff
@@ -0,0 +1,72 @@
+# Voluntary exits
+
+A validator may chose to voluntary stop performing duties (proposing blocks and attesting to blocks) by submitting
+a voluntary exit transaction to the beacon chain.
+
+A validator can initiate a voluntary exit provided that the validator is currently active (has not been slashed) and has been active for at least 256 epochs (~27 hours) since it has been activated.
+
+> Note: After initiating a voluntary exit, the validator will have to keep performing duties until it has successfully exited to avoid penalties.
+
+It takes at minimum 5 epochs (32 minutes) for a validator to exit after initiating a voluntary exit.
+This number can be much higher depending on how many other validators are in queue to exit.
+
+## Withdrawal of exited funds
+
+Even though users can perform a voluntary exit in phase 0, they **cannot withdraw their exited funds until phase 2**.
+This implies that the staked funds are effectively **frozen** until withdrawals are enabled in future phases.
+
+To understand the phased rollout strategy for Eth2, please visit <https://ethereum.org/en/eth2/#roadmap>.
+
+
+
+## Initiating a voluntary exit
+
+In order to initiate a exit, users can use the `lighthouse account validator exit` command.
+
+The `--validator` flag is used to specify the validator for which the user wants to initiate an exit transaction.
```

Comment:
If we don't just use an EIP-2335 keystore path instead, I think we should specify how the value of this flag identifies a validator.

If we *do* just use an EIP-2335 keystore path, you can ignore this comment.